core/templates/dev/head/pages/about/About.js

```diff
@@ -91,5 +91,9 @@ oppia.controller('About', [
     $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
     $scope.aboutPageMascotImgUrl = UrlInterpolationService.getStaticImageUrl(
       '/general/about_page_mascot.png');
+
+    // Set page title
+    var pageTitle = 'About us - Oppia';
+    PageTitleService.setPageTitle(pageTitle);
```

Comment:
I just realized that if the title is not dynamic we might not want to set it through Angular. Because, when we move to static pages we will have something like:

```
<head>
  <!-- webpack import of common head stuff -->
  <meta>custom meta tag for this page</meta>
  <title>static string title</title>
<head>
```

And I think that it might be useful to keep as much stuff as we can in the HTML.

Reply:
Okay, meaning that all pages with static title will have to be removed from the list

Reply:
Yes